concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1393,6 +1395,11 @@ jobs:
 
                   echo "Generating manifests"
 
+                  if [ "${ENABLE_DATADOG}" = "true" ] ; then
+                     MANIFEST_STUBS="${MANIFEST_STUBS}
+                                    ./paas-cf/manifests/cf-manifest/stubs/datadog-nozzle.yml"
```

Comment:
why new directory and naming scheme? Why not call this e.g. manifests/cf-manifest/060-datadog-nozzle.yml ?